### PR TITLE
add description about dependencies projects #1982

### PR DESCRIPTION
**Pull request by bttanakagn**
Please review #1982.

### CODE REVIEW

File: source/Overview/FrameworkStack.rst [hunk 1]
```diff
@@ -705,20 +705,72 @@ version 5.1.0.RELEASEで利用するOSSの一覧を以下に示す。
...
+      - terasoluna-gfw-webプロジェクトが提供する機能を使用する場合の依存関係定義を提供する。
+      - 無
+      - 有
+      - 有
+    * - \ (21)
+      - terasoluna-gfw-web-jsp-dependencies
+      - terasoluna-gfw-web-jspプロジェクトが提供する機能を使用する場合の依存関係定義を提供する。
+      - 無
+      - 有
+      - 有
```
Comment (btyoshikawaa) [inline source/Overview/FrameworkStack.rst]: @bttanakagn 
I think, it is better to rearrange order. 
For example...
* `common-libraries` -> `dependencies` -> `parent`
* `parent` -> `common-libraries` -> `dependencies`
* [ `common` -> `common-depnedencies` ] *n ...

File: source/Overview/FrameworkStack.rst [hunk 2]
```diff
@@ -705,20 +705,72 @@ version 5.1.0.RELEASEで利用するOSSの一覧を以下に示す。
...
-      - 有*3
-      - 有*3
+      - 有*4
+      - 有*4
+    * - \ (15)
+      - terasoluna-gfw-common-libraries
+      - 共通ライブラリのうち、Javaソースコードを含むプロジェクトの構成を定義する。
+      - 無
+      - 有*4
+      - 有*4
```
Comment (btyoshikawaa) [inline source/Overview/FrameworkStack.rst]: - [ ] Are users necessary to aware `terasoluna-gfw-common-libraries` and `terasoluna-gfw-dependencies` ? 
I want information that the user can determine.

File: source/Overview/FrameworkStack.rst [hunk 3]
```diff
@@ -625,10 +625,10 @@ version 5.1.0.RELEASEで利用するOSSの一覧を以下に示す。
       - シングルプロジェクト構成のブランクプロジェクト組込
     * - \ (1)
       - terasoluna-gfw-common
-      - Webに依存しない汎用的に使用できる機能と依存関係定義を提供する。
-      - 有
-      - 有
+      - Webに依存しない汎用的に使用できる機能を提供する。
       - 有
+      - 有*1
+      - 有*1
```
Comment (btyoshikawaa) [inline source/Overview/FrameworkStack.rst]: - [ ] I want information that the users should use the `common-dependencies` If they use this.

File: source/Overview/FrameworkStack.rst (hunk 2 above)
Comment (bttanakagn) [inline source/Overview/FrameworkStack.rst]: @btyoshikawaa 
I added information that the user do not have to add this artifact to `pom.xml`. 4c1c10f559c58dc66f62019e1dffd21717c550cf.

File: source/Overview/FrameworkStack.rst (hunk 1 above)
Comment (bttanakagn) [inline source/Overview/FrameworkStack.rst]: @btyoshikawaa 
I rearranged it like the way of [ `common` -> `common-depnedencies` ] *n ... 1fe050c1bb97edf24cb6bae288e0f80552acc650

File: source/Overview/FrameworkStack.rst (hunk 3 above)
Comment (bttanakagn) [inline source/Overview/FrameworkStack.rst]: @btyoshikawaa 
I added the information. 905a3ca8ed00bb8cbb3eb83809a1b88d3d36bb8f

**Comment by btyoshikawaa** [comment]
@bttanakagn 
OK, and I will merge this after you rebase.

**Pull request merged**